minpy/array.py

```diff
@@ -572,9 +578,7 @@ def get_val(x, mutate):
         }
         # Call the real function with raw value.
         if self.type == ArrayType.MXNET:
-            # Currently all MXNet function call will be performed on GPU 0.
-            with mxnet.gpu(0) as ctx:
```

Comment:
I do some test in mxnet:

```
import mxnet as mx
x = mx.nd.ones((2, 3))
# x.context == cpu(0)
y = mx.nd.ones((3, 4))
# y.context == cpu(0)
with mx.Context(mx.gpu(0)):
    z = mx.nd.dot(x, y)
    print(z.context)
```
This code snippet above will return `cpu(0)`. So I wonder whether this line have effect on the final result if the context of arguments has already beed fixed?